test/test_container_application.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+
+from container_ci_suite.container_lib import ContainerTestLib
+from container_ci_suite.engines.podman_wrapper import PodmanCLIWrapper
+
+from conftest import VARS
+
+
+class TestRedisApplicationContainer:
+    def setup_method(self):
+        self.s2i_app = ContainerTestLib(image_name=VARS.IMAGE_NAME, s2i_image=True)
+
+    def teardown_method(self):
+        self.s2i_app.cleanup()
+
+    @pytest.mark.parametrize(
+        "password,test_name",
+        [
+            ("pass", "no_root"),
+            ("", "no_pass"),
+            ("pass", "no_pass_altuid"),
+            ("pass", "no_root_altuid"),
+        ]
+    )
+    def test_run_app_test(self, password, test_name):
+        """
+        Test checks if valkey server works with different settings.
+        Like 'No root' user with password, 'w/o password'
+        and with different user and password
+        """
+        container_arg = f"-e REDIS_PASSWORD={password}" if password != "" else ""
```

Comment:
In Python, empty strings are falsy, so this can be simplified.


```suggestion
        container_arg = f"-e REDIS_PASSWORD={password}" if password else ""
```